configs/katello/4.5.yaml

```diff
@@ -0,0 +1,146 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.5:
+    :redmine_version_id: 1543
+:prior_releases:
+  :4.4.0.2:
+    :redmine_version_id: 1590
+  :4.4.0.1:
+    :redmine_version_id: 1586
+  :4.4.0:
+    :redmine_version_id: 1517
+:code_name: Old Monk
+:mash_scripts:
+  - katello-4.5-rhel7
+  - katello-4.5-thirdparty-candlepin-rhel7
+  - katello-4.5-el8
+  - katello-candlepin-4.5-el8
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-4.5
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/3.3
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 1.5.z
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 4.0.1
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :katello-certs-tools:
+    :branch: 2.9.0
+    :repo: https://github.com/Katello/katello-certs-tools.git
+  :foreman_virt_who_configure:
+    :branch: 0.5.8
```

Comment:
looks like we are still on 0.5.6 for this guy so lets change this to 0.5.7

Reply:
hmm I wonder if this field is ignored. I mean look at -> https://github.com/theforeman/tool_belt/blob/master/configs/katello/4.4.yaml#L41

Reply:
Lets ignore and merge :shipit: